src/components/CompareResults/ResultsTable.tsx

```diff
@@ -30,45 +28,44 @@ function ResultsTable(props: ResultsTableProps) {
   const styles = {
     tableContainer: style({
       backgroundColor: themeColor100,
+      borderCollapse: 'separate',
+      borderSpacing: `0 ${Spacing.Small}px`,
       marginTop: Spacing.Large,
-      $nest: {
-        '.MuiTable-root': {
-          borderSpacing: `0 ${Spacing.Small}px`,
-          borderCollapse: 'separate',
-          paddingBottom: Spacing.Large,
-        },
-      },
+      paddingBottom: Spacing.Large,
     }),
   };
 
   return (
-    <TableContainer
-      component={Paper}
+    <Paper
       className={styles.tableContainer}
       data-testid='results-table'
       sx={customStyles}
+      role='table'
     >
       {loading ? (
         <Box display='flex' justifyContent='center' alignItems='center'>
           <CircularProgress />
         </Box>
       ) : (
-        <Table>
+        <div>
           <TableHeader themeMode={themeMode} />
           {processedResults.map((res, index) => (
-            <TableContent
-              themeMode={themeMode}
-              key={index}
-              header={res.revisionHeader}
-              results={res.value}
-            />
+            <div key={index}>
```

Comment:
Is this div useful? I see that the TableContent component itself also has a div already as its root element.